components/MdInput.test.js

```diff
@@ -40,8 +41,61 @@ describe('MdInput Component', () => {
 
     const textbox = queryByTestId('textbox')
     const markdown = queryByTestId('markdown')
-    expect(textbox).toBeNull()
+    expect(textbox.classList.contains('d-none')).toBe(true)
     expect(markdown).toBeTruthy()
     expect(container).toMatchSnapshot()
   })
+
+  test('Should save cursor state going to and back from preview', () => {
+    render(<TestComponent />)
+    const previewButton = screen.getByRole('button', { name: 'Preview' })
+    const writeButton = screen.getByRole('button', { name: 'Write' })
+    const textbox = screen.getByRole('textbox')
+
+    userEvent.click(textbox)
+    userEvent.type(textbox, 'Hello,{enter}World!')
+    expect(textbox.value).toBe('Hello,\nWorld!')
```

Comment:
Also from `jest-dom`, you could use [`toHaveValue()`](https://github.com/testing-library/jest-dom#tohavevalue) here.

Reply:
Ok nice to know! I'll add it when I get back to my computer